lib/dao.js

```diff
@@ -2356,6 +2559,200 @@ DataAccessObject.prototype.unsetAttribute = function unsetAttribute(name, nullif
 };
 
 /**
+ * Replace set of attributes.
+ * Performs validation before replacing.
+ *
+ * @trigger `validation`, `save` and `update` hooks
+ * @param {Object} data Data to replace
+ * @param {Object} [options] Options for replaceAttributes
+ * @param {Function} cb Callback function called with (err, instance)
+ */
+
+DataAccessObject.prototype.replaceAttributes = function replaceAttributes(data, options, cb) {
```

Comment:
Can we rename it to `replace` as `replaceAttributes` hints only the attrs will be replaced?